isis/scripts/zenodo_order.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+"""
+This program reads the .zenodo.json file finds the
+designated first author, puts them first, and then puts the
+remainder in alphabetical order.
+"""
+
+import argparse
+import json
+import sys
+
+from operator import itemgetter
+
+# This is nominally the ISIS project lead or whomever should be
+# the first author in the Zenodo authors listing.
+first_author = "Laura, Jason"
```

Comment:
This should be after the license comment.

I also feel like this is an argument, but not one I want to mess with often. Making it something with a default value is kind of odd. Is there a better way to handle this?

Reply:
If we knew more about what keys were allowed in the .zenodo.json file (see previous discussions on the woeful lack of documentation), there might be a way to define a key/value pair in one of the creator records to designate the project lead, and then we wouldn't have to force it here.

Aside from that, sure making it a defaulted argument is fine, too.  I'll work it.